roles/epel/tasks/main.yml

```diff
@@ -0,0 +1,12 @@
+---
+- name: Check if EPEL is installed
+  stat:
+    path: /etc/yum.repos.d/epel.repo
+  register: epel_installed
+  when: ansible_distribution == "RedHat"
+
+- name: Install EPEL repo
+  yum:
+    name: "https://dl.fedoraproject.org/pub/epel/epel-release-latest-{{ ansible_distribution_major_version }}.noarch.rpm"
+    state: present
+  when: not epel_installed.stat.exists and ansible_distribution == "RedHat"
```

Comment:
Fixed in https://github.com/StackStorm/ansible-st2/pull/66/commits/da000f663a3626cb5ddd02516443e5bd04332469